src/tallies/tally.cpp

```diff
@@ -381,6 +395,12 @@ void Tally::add_filter(Filter* filter)
     energyout_filter_ = filters_.size();
   } else if (dynamic_cast<const DelayedGroupFilter*>(filter)) {
     delayedgroup_filter_ = filters_.size();
+  } else if ((filter->type() == FilterType::CELL) &&
+             !(filter->type() == FilterType::CELLBORN) &&
+             !(filter->type() == FilterType::CELLFROM)) {
```

Comment:
Since `FilterType` is just an enum, there is no way it can simultaneously be equal to `CELL` and something else. Just checking for equality with `FilterType::CELL` should be sufficient.

Reply:
yes